.swiftformat

```diff
@@ -0,0 +1,16 @@
+# file options
+
+--exclude Tests/LinuxMain.swift,Package.swift
+
+# format options
+--swiftversion 4.2
```

Comment:
Does 4.0 work here? Our `Package.swift` specifies 4.0

Reply:
I'm not sure but I think it will work.

I tried adding matrix to run with Swift 4.0 on Travis🙂

Reply:
CI failed on 4.0 and 4.1. 
It seems to be required 4.2 on [RequestKit](https://github.com/nerdishbynature/RequestKit/blob/master/Package.swift#L1)🤔